migration-runner.sh

```diff
@@ -43,83 +18,65 @@ source steps/check-args.sh
 if [ -z "$ENV" ]; then
     echo "[*] Usage: $0 -e [env]"
     echo "[*] Usage: $1 -o [operation]"
-    echo "[*] Usage: $2 -db [dbtype]"
     echo "    Mandatory flags"
     echo "    -e [env]"
     echo "    -o [operation]"
-    echo "    -db [dbtype]"
     echo
     echo "    Optional flags"
     echo "    -j [jurisdiction]"
-    echo "    -d [from_date]"
-    echo "    -f [definition_store_snapshot]"
-    echo "    -t [data_store_snapshot]"
     echo "    -s [staging_table]"
     echo
     exit 1
 fi
 
-case "$ENV$OPERATION$DBTYPE" in
-    localtakesnapshotsrealtime)
-        source steps/get-database-credentials.sh
-        source steps/take-database-snapshots.sh
-#        source steps/start-local-postgres-server.sh
-#        source steps/load-database-snapshots.sh
-#        source steps/get-document-keys.sh
-#        source steps/export-document-keys.sh
-#        source steps/import-document-keys-to-data-store.sh
-#        source steps/migrate-staging-table.sh
-#        source steps/clean-up.sh
+case "$ENV$OPERATION" in
```

Comment:
can we just say $OPERATION and include only two options underneath .. so that they don't need to pass env at all ??

Reply:
If you agree please update README